Makefile

```diff
@@ -22,7 +22,6 @@ FLAKE8_EXCLUDE=.git,*raw_sql.py
 
 .PHONY: flake8
 flake8:
-	@flake8 --exclude=$(FLAKE8_EXCLUDE) --ignore=E722,E501 tcms *.py
```

Comment:
oops, almost missed this one. Deleting the line disables flake8 for everything in the tcms/ directory. Remove onky the ignore flag.

Reply:
Done